docs/guides/mutations.md

```diff
@@ -324,5 +324,5 @@ We also have an extensive [offline example](../examples/offline) that covers bot
 
 ## Further reading
 
-For more information about mutations, have a look at [#12: Mastering Mutations in React Query](../community/tkdodos-blog#12-mastering-mutations-in-react-query) from
+For more information about mutations, have a look at [#12: Mastering Mutations in React Query](https://tkdodo.eu/blog/mastering-mutations-in-react-query) from
```

Comment:
thanks, but the link should go to the `community` resources that eventually link here:

https://tanstack.com/query/v4/docs/community/tkdodos-blog#12-mastering-mutations-in-react-query

can you fix that, and maybe all other links that go to the community resources? thanks.

Reply:
okay, i'll do that

Reply:
I'm sorry that I wasn't clear enough :( the links need to stay relative links please, otherwise, we have to change them with every majore release...